content/post/career-ladders-part-1.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Great first post!

To play devils advocate -- is there any good reason not to create a career ladder? Can you imagine someone disagreeing with anything here? If yes, could be worth addressing. If no, then ignore.